open-machine-learning-jupyter-book/assignments/python-programming-basics.ipynb

```diff
@@ -1,59 +1,215 @@
-{

```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #48.    
</pre><p>You need to add a acknowledge section similar to the book main content. And we want to thanks to the code author here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>